jekyll/assets/js/main.js

```diff
@@ -261,10 +258,81 @@ $( document ).ready(function() {
     $("nav.sidebar").toggleClass("open");
   });
 
+  var tooltip = document.querySelector(".server-version-popover");
+
   // Give article headings direct links to anchors
   $("article h2, article h3, article h4, article h5, article h6").filter("[id]").each(function () {
     $(this).append('<a href="#' + $(this).attr("id") + '"><i class="fa fa-link"></i></a>');
   });
+
+  var showEvents = ['mouseover', 'hover', 'mouseenter', 'focus'];
+  var hideEvents = [
+    'mouseout', 
+    'mouseleave', 
+    'blur'
+  ];
+  var clickEvents = ['click'];
+
+  var makePopper = (icon) => Object.assign(icon, {
+    show() {
+      tooltip.setAttribute('data-show', '');
+      // change tooltip text based on current button popover.
+      tooltip.innerHTML = "Copy link<div id='arrow' data-popper-arrow></div>"
+      icon.instance = Popper.createPopper(icon, tooltip, {
+        modifiers: [
+          {name: 'offset',
+          options: {
+            offset: [0, 8],
+          },
+          },
+        ],
+      });
+    },
+    copy(event) {
+      event.preventDefault();
+      navigator.clipboard
+        .writeText(event.target.href)
+        .then(() => {
+          tooltip.innerHTML = "Copied!<div id='arrow' data-popper-arrow></div>";
+
+          document.querySelector(this.getAttribute('href')).scrollIntoView({
+            behavior: 'smooth'
+          });
+          window.history.pushState({}, document.title, event.target.href);
+        });
+    },
+    hide() {
+      tooltip.removeAttribute('data-show');
+      if (icon.instance) {
+        icon.instance.destroy();
+        icon.instance = null;
+      }
+    }
+  });
+
+  window.OptimizelyClient.getVariationName({
+    experimentKey: 'dd_share_section_icon_test',
+    groupExperimentName: 'q3_fy22_docs_disco_experiment_group_test'
+  }).then(variation => {
+    if (variation === 'treatment') {
+      document.querySelectorAll('.fa-link').forEach(icon => {
+        makePopper(icon);
+
+        showEvents.forEach(event => {
+          icon.parentElement.addEventListener(event, icon.show);
+        });
+
+        hideEvents.forEach(event => {
+          icon.parentElement.addEventListener(event, icon.hide);
+        });
+
+        clickEvents.forEach(event => {
+          icon.parentElement.addEventListener(event, icon.copy);
+        });
+      });
+    }
+  });
+
```

Comment:
Maybe we could group all experiments into a single folder for easier graduation / cleanup in the future?

Reply:
I think we can start doing that once we get webpack configured.

Reply:
@teesloane @nramabad 🔵  list please 🙏